src/backend/common/value.cpp

```diff
@@ -85,6 +85,10 @@ Value &Value::operator=(const Value &other) {
           Varlen *src_sref = *reinterpret_cast<Varlen *const *>(other.m_data);
           Varlen *new_sref = Varlen::Clone(*src_sref, nullptr);
 
+          // TODO: Fix memory leak problem.
```

Comment:
If I released the string using the way in destructor, basically I could pass small unit test, but there were somewhere else broken. It may change a lot. So it is better to fix this one in another PR.